### PR TITLE
new icon: nuxtjs (original, original-wordmark, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by TheFoxarmy**
Added NuxtJS SVG icons from the official website: https://nuxtjs.org/design
Created the wordmark version based on their homepage icon.
Addresses issues #736

### CODE REVIEW

**Comment by Thomas-Boi** [comment]
Hi @TheFoxarmy,

Welcome to Devicon 😄. Before we continue, there's something we need to change for this PR. Currently, it includes both the changes for `nuxtjs` and `socketio`. Please make it so that only `nuxtjs` changes (`devicon.json`, SVG files) are present in this PR please.

**Comment by TheFoxarmy** [comment]
Thank you, I just noticed this! I accidentally made both additions on master. I moved the last commit, The socketio one to its own branch, and committed that for the other PR, but for this, I just need to remove the latest commit on master.

**Comment by TheFoxarmy** [comment]
I have converted all icons to use 128x `viewbox` and removed the `socketio` changes from the master branch

File: icons/nuxtjs/nuxtjs-original-wordmark.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....4 0 0 1-9.8 0 6.2 6.2 0 0 1-2-4.5v-75.4h-23.8c-1.7 0-3.3-0.5-4.6-1.7a5.9 5.9 0 0 1-1.9-4.5c0-1.6 0.6-3.2 1.8-4.3 1.3-1.2 3-1.8 4.6-1.7h61.4c1.7 0 3.4 0.5 4.7 1.7z" fill="#2e495e"/><path d="m412.7 93.6c-4.3-2.4-8-6-10.5-10.3-0.7-1-1.1-2.3-1.2-3.6 0-1.7 0.7-3.3 2-4.4 1.1-1.2 2.6-1.8 4.2-2 2.4 0.3 4.6 1.5 6 3.4 3 5 8.4 8 14.1 7.8a17.2 17.2 0 0 0 17.7-17.2v-58.8a6 6 0 0 1 2-4.6c1.3-1.2 3-1.9 4.7-1.9 1.8 0 3.5 0.6 4.8 1.8a6.3 6.3 0 0 1 1.8 4.6v59a29 29 0 0 1-15 26 31.2 31.2 0 0 1-30.5 0.2zm92.5 0.4c-5.3-2-10.2-5-14.4-9a7.2 7.2 0 0 1-2.8-5.4 6 6 0 0 1 2-4.4 6 6 0 0 1 4.3-1.9c1.5 0 2.8 0.5 3.9 1.5 3.2 3 7 5.5 11 7.3 4.2 1.6 8.6 2.4 13 2.4 5 0.1 10-1.2 14.3-4 3.5-2 5.8-5.7 5.8-9.8 0.1-4.3-2.1-8.4-5.8-10.8a52 52 0 0 0-17.4-6.3c-9.2-2-16.3-5-21.3-9.3s-7.4-10.3-7.4-17.9c-0.1-5 1.3-9.9 4.2-14 2.9-4 6.8-7.2 11.4-9.2a45.5 45.5 0 0 1 32.4-0.5 33 33 0 0 1 12 7.1c1.8 1.4 3 3.5 3 5.7a6 6 0 0 1-1.8 4.3 5.7 5.7 0 0 1-4.4 2c-1 0-2.2-0.4-3-1.1a32.1 32.1 0 0 0-21.4-8.2c-5-0.3-10 1-14.3 3.6a11.9 11.9 0 0 0 0 19.8 50 50 0 0 0 15.8 6c5.9 1.1 11.6 3 17.1 5.3 4.2 1.9 7.8 4.8 10.3 8.6 2.6 4.4 3.8 9.3 3.6 14.4 0.1 5-1.4 9.8-4.3 13.8-3 4.2-7.2 7.4-12 9.4a49.8 49.8 0 0 1-33.9 0.6z" fill="#00c48d"/></g></g></svg>
```
Comment (Panquesito7) [inline icons/nuxtjs/nuxtjs-original-wordmark.svg:1]:
```suggestion
<svg version="1.1" width="128" height="128" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><g transform="matrix(.51034 0 0 .51034 -63.584 40.014)"><g transform="matrix(.13925 0 0 .13925 124.51 23.118)"><path d="m139 330-1-2c-2-4-2-8-1-13h-108l160-284 67 121 22-16-67-121c-1-2-9-14-22-14-6 0-15 2-22 15l-162 287c-1 3-8 16-2 27 4 6 10 12 24 12h136c-14 0-21-6-24-12z" fill="#00c58e"/><path d="m447 304-130-234c-2-2-9-15-22-15-6 0-15 3-22 15l-17 28v54l39-67 129 230h-49a23 23 0 0 1-2 14l-1 1c-6 11-21 12-23 12h76c3 0 17-1 24-12 3-5 5-14-2-26z" fill="#108775"/><path d="m376 330v-1l1-2c1-4 2-8 1-12l-4-12-102-178-15-27h-1l-15 27-102 178-4 12a24 24 0 0 0 2 15c4 6 10 12 24 12h190c3 0 18-1 25-12zm-120-178 93 163h-186z" fill="#2f495e"/></g><g transform="matrix(.31945 0 0 .31945 198.01 39.602)" fill-rule="evenodd"><path d="m74.4 3.7c1 1.1 1.7 2.6 1.6 4.2v81.5c0 1.7-0.6 3.4-1.8 4.7a6.2 6.2 0 0 1-4.6 1.9c-1 0-2-0.2-3-0.5a5 5 0 0 1-2.1-1.4l-52.8-68.7v64.6a5.7 5.7 0 0 1-6 5.9c-1.5 0-3-0.6-4-1.7a5.8 5.8 0 0 1-1.7-4.1v-81.7c0-1.7 0.7-3.4 2-4.6a6 6 0 0 1 4.3-1.8c1 0 2.1 0.2 3 0.6 1 0.3 1.8 1 2.4 1.7l52.5 68.9v-65.3a5.7 5.7 0 0 1 6-5.9c1.5 0 3 0.5 4 1.7zm108.7 0.2c1.1 1.3 1.8 3 1.7 4.6v51.6c0.1 6.6-1.5 13.2-4.7 19-3 5.6-7.5 10.2-13 13.3-5.6 3.3-12 5-18.6 4.9-6.5 0-13-1.6-18.8-4.8a34 34 0 0 1-13-13.2 38.6 38.6 0 0 1-4.7-19.1v-51.6c0-1.7 0.6-3.4 2-4.6a6.7 6.7 0 0 1 9.3 0c1.2 1.2 1.9 2.8 1.9 4.6v51.4c-0.1 4.4 1 8.7 3.2 12.6a23.2 23.2 0 0 0 20.1 11.7 24 24 0 0 0 24.1-24.3v-51.4c0-1.7 0.5-3.4 1.6-4.7 1.2-1.3 2.8-2 4.5-1.9 1.7 0 3.3 0.7 4.4 1.9zm103.9 81.8a6.3 6.3 0 0 1 1.1 3.8c0.1 1.7-0.6 3.4-1.9 4.6a6.4 6.4 0 0 1-4.3 1.7c-2.1 0.1-4.1-1-5.3-2.7l-26-34.8-25.6 34.8a6.2 6.2 0 0 1-5.2 2.8c-1.6 0-3-0.6-4.1-1.7-1-1-1.7-2.4-1.7-3.8 0-1.6 0.6-3.2 1.6-4.4l27.6-37.5-27.7-37.2a6.2 6.2 0 0 1 0.5-8.4 6.3 6.3 0 0 1 7.4-1.2 6 6 0 0 1 2.4 2l25.8 34.6 25.4-34.3c1.1-1.7 3-2.8 5.1-3 1.6 0 3.2 0.8 4.3 2 1.1 1 1.8 2.5 1.9 4 0 1.5-0.5 2.8-1.4 4l-28 37.2 28 37.5zm96.5-82c1.2 1.1 2 2.8 1.9 4.5 0 1.7-0.7 3.3-2 4.4-1.2 1.1-2.9 1.7-4.6 1.6h-23.8v75.4c0 1.7-0.6 3.3-1.9 4.5a7.4 7.4 0 0 1-9.8 0 6.2 6.2 0 0 1-2-4.5v-75.4h-23.8c-1.7 0-3.3-0.5-4.6-1.7a5.9 5.9 0 0 1-1.9-4.5c0-1.6 0.6-3.2 1.8-4.3 1.3-1.2 3-1.8 4.6-1.7h61.4c1.7 0 3.4 0.5 4.7 1.7z" fill="#2e495e"/><path d="m412.7 93.6c-4.3-2.4-8-6-10.5-10.3-0.7-1-1.1-2.3-1.2-3.6 0-1.7 0.7-3.3 2-4.4 1.1-1.2 2.6-1.8 4.2-2 2.4 0.3 4.6 1.5 6 3.4 3 5 8.4 8 14.1 7.8a17.2 17.2 0 0 0 17.7-17.2v-58.8a6 6 0 0 1 2-4.6c1.3-1.2 3-1.9 4.7-1.9 1.8 0 3.5 0.6 4.8 1.8a6.3 6.3 0 0 1 1.8 4.6v59a29 29 0 0 1-15 26 31.2 31.2 0 0 1-30.5 0.2zm92.5 0.4c-5.3-2-10.2-5-14.4-9a7.2 7.2 0 0 1-2.8-5.4 6 6 0 0 1 2-4.4 6 6 0 0 1 4.3-1.9c1.5 0 2.8 0.5 3.9 1.5 3.2 3 7 5.5 11 7.3 4.2 1.6 8.6 2.4 13 2.4 5 0.1 10-1.2 14.3-4 3.5-2 5.8-5.7 5.8-9.8 0.1-4.3-2.1-8.4-5.8-10.8a52 52 0 0 0-17.4-6.3c-9.2-2-16.3-5-21.3-9.3s-7.4-10.3-7.4-17.9c-0.1-5 1.3-9.9 4.2-14 2.9-4 6.8-7.2 11.4-9.2a45.5 45.5 0 0 1 32.4-0.5 33 33 0 0 1 12 7.1c1.8 1.4 3 3.5 3 5.7a6 6 0 0 1-1.8 4.3 5.7 5.7 0 0 1-4.4 2c-1 0-2.2-0.4-3-1.1a32.1 32.1 0 0 0-21.4-8.2c-5-0.3-10 1-14.3 3.6a11.9 11.9 0 0 0 0 19.8 50 50 0 0 0 15.8 6c5.9 1.1 11.6 3 17.1 5.3 4.2 1.9 7.8 4.8 10.3 8.6 2.6 4.4 3.8 9.3 3.6 14.4 0.1 5-1.4 9.8-4.3 13.8-3 4.2-7.2 7.4-12 9.4a49.8 49.8 0 0 1-33.9 0.6z" fill="#00c48d"/></g></g></svg>

```

File: icons/nuxtjs/nuxtjs-original.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...9.267 108.97-0.28372-0.56744c-0.56742-1.1349-0.56742-2.2697-0.28372-3.6883h-30.641l45.395-80.575 19.009 34.33 6.2417-4.5395-19.009-34.33c-0.28372-0.56742-2.5534-3.972-6.2417-3.972-1.7023 0-4.2557 0.56742-6.2417 4.2557l-45.961 81.426c-0.28372 0.85115-2.2697 4.5395-0.56742 7.6603 1.1349 1.7023 2.8372 3.4046 6.8091 3.4046h38.585c-3.972 0-5.958-1.7023-6.8091-3.4046z" fill="#00c58e" stroke-width=".28372"/><path d="m126.65 101.59-36.883-66.389c-0.56742-0.56742-2.5534-4.2557-6.2417-4.2557-1.7023 0-4.2557 0.85114-6.2417 4.2557l-4.8231 7.944v15.321l11.065-19.009 36.599 65.254h-13.902a6.5254 6.5254 0 0 1-0.56744 3.972l-0.28371 0.28374c-1.7023 3.1208-5.958 3.4046-6.5254 3.4046h21.562c0.85114 0 4.8231-0.28371 6.8091-3.4046 0.85114-1.4186 1.4185-3.972-0.56742-7.3765z" fill="#108775" stroke-width=".28372"/><path d="m106.51 108.97v-0.28374l0.28372-0.56741c0.28371-1.1349 0.56742-2.2697 0.28371-3.4046l-1.1349-3.4046-28.938-50.501-4.2557-7.6603h-0.28372l-4.2557 7.6603-28.938 50.501-1.1349 3.4046a6.8091 6.8091 0 0 0 0.56743 4.2557c1.1349 1.7023 2.8372 3.4046 6.8091 3.4046h53.906c0.85114 0 5.1068-0.28371 7.0929-3.4046zm-34.046-50.501 26.386 46.245h-52.771z" fill="#2f495e" stroke-width=".28372"/></svg>
```
Comment (Panquesito7) [inline icons/nuxtjs/nuxtjs-original.svg:1]:
```suggestion
<svg version="1.1" width="128" height="128" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path d="m39.267 108.97-0.28372-0.56744c-0.56742-1.1349-0.56742-2.2697-0.28372-3.6883h-30.641l45.395-80.575 19.009 34.33 6.2417-4.5395-19.009-34.33c-0.28372-0.56742-2.5534-3.972-6.2417-3.972-1.7023 0-4.2557 0.56742-6.2417 4.2557l-45.961 81.426c-0.28372 0.85115-2.2697 4.5395-0.56742 7.6603 1.1349 1.7023 2.8372 3.4046 6.8091 3.4046h38.585c-3.972 0-5.958-1.7023-6.8091-3.4046z" fill="#00c58e" stroke-width=".28372"/><path d="m126.65 101.59-36.883-66.389c-0.56742-0.56742-2.5534-4.2557-6.2417-4.2557-1.7023 0-4.2557 0.85114-6.2417 4.2557l-4.8231 7.944v15.321l11.065-19.009 36.599 65.254h-13.902a6.5254 6.5254 0 0 1-0.56744 3.972l-0.28371 0.28374c-1.7023 3.1208-5.958 3.4046-6.5254 3.4046h21.562c0.85114 0 4.8231-0.28371 6.8091-3.4046 0.85114-1.4186 1.4185-3.972-0.56742-7.3765z" fill="#108775" stroke-width=".28372"/><path d="m106.51 108.97v-0.28374l0.28372-0.56741c0.28371-1.1349 0.56742-2.2697 0.28371-3.4046l-1.1349-3.4046-28.938-50.501-4.2557-7.6603h-0.28372l-4.2557 7.6603-28.938 50.501-1.1349 3.4046a6.8091 6.8091 0 0 0 0.56743 4.2557c1.1349 1.7023 2.8372 3.4046 6.8091 3.4046h53.906c0.85114 0 5.1068-0.28371 7.0929-3.4046zm-34.046-50.501 26.386 46.245h-52.771z" fill="#2f495e" stroke-width=".28372"/></svg>

```

File: icons/nuxtjs/nuxtjs-plain-wordmark.svg [hunk 3]
```diff
@@ -0,0 +1 @@
....4c0 1.7-0.6 3.3-1.9 4.5a7.4 7.4 0 0 1-9.8 0 6.2 6.2 0 0 1-2-4.5v-75.4h-23.8c-1.7 0-3.3-0.5-4.6-1.7a5.9 5.9 0 0 1-1.9-4.5c0-1.6 0.6-3.2 1.8-4.3 1.3-1.2 3-1.8 4.6-1.7h61.4c1.7 0 3.4 0.5 4.7 1.7z"/><path d="m412.7 93.6c-4.3-2.4-8-6-10.5-10.3-0.7-1-1.1-2.3-1.2-3.6 0-1.7 0.7-3.3 2-4.4 1.1-1.2 2.6-1.8 4.2-2 2.4 0.3 4.6 1.5 6 3.4 3 5 8.4 8 14.1 7.8a17.2 17.2 0 0 0 17.7-17.2v-58.8a6 6 0 0 1 2-4.6c1.3-1.2 3-1.9 4.7-1.9 1.8 0 3.5 0.6 4.8 1.8a6.3 6.3 0 0 1 1.8 4.6v59a29 29 0 0 1-15 26 31.2 31.2 0 0 1-30.5 0.2zm92.5 0.4c-5.3-2-10.2-5-14.4-9a7.2 7.2 0 0 1-2.8-5.4 6 6 0 0 1 2-4.4 6 6 0 0 1 4.3-1.9c1.5 0 2.8 0.5 3.9 1.5 3.2 3 7 5.5 11 7.3 4.2 1.6 8.6 2.4 13 2.4 5 0.1 10-1.2 14.3-4 3.5-2 5.8-5.7 5.8-9.8 0.1-4.3-2.1-8.4-5.8-10.8a52 52 0 0 0-17.4-6.3c-9.2-2-16.3-5-21.3-9.3s-7.4-10.3-7.4-17.9c-0.1-5 1.3-9.9 4.2-14 2.9-4 6.8-7.2 11.4-9.2a45.5 45.5 0 0 1 32.4-0.5 33 33 0 0 1 12 7.1c1.8 1.4 3 3.5 3 5.7a6 6 0 0 1-1.8 4.3 5.7 5.7 0 0 1-4.4 2c-1 0-2.2-0.4-3-1.1a32.1 32.1 0 0 0-21.4-8.2c-5-0.3-10 1-14.3 3.6a11.9 11.9 0 0 0 0 19.8 50 50 0 0 0 15.8 6c5.9 1.1 11.6 3 17.1 5.3 4.2 1.9 7.8 4.8 10.3 8.6 2.6 4.4 3.8 9.3 3.6 14.4 0.1 5-1.4 9.8-4.3 13.8-3 4.2-7.2 7.4-12 9.4a49.8 49.8 0 0 1-33.9 0.6z"/></g></g></svg>
```
Comment (Panquesito7) [inline icons/nuxtjs/nuxtjs-plain-wordmark.svg:1]:
```suggestion
<svg version="1.1" width="128" height="128" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><g transform="matrix(.51034 0 0 .51034 -63.584 40.014)"><g transform="matrix(.13925 0 0 .13925 124.51 23.118)"><path d="m139 330-1-2c-2-4-2-8-1-13h-108l160-284 67 121 22-16-67-121c-1-2-9-14-22-14-6 0-15 2-22 15l-162 287c-1 3-8 16-2 27 4 6 10 12 24 12h136c-14 0-21-6-24-12z"/><path d="m447 304-130-234c-2-2-9-15-22-15-6 0-15 3-22 15l-17 28v54l39-67 129 230h-49a23 23 0 0 1-2 14l-1 1c-6 11-21 12-23 12h76c3 0 17-1 24-12 3-5 5-14-2-26z"/><path d="m376 330v-1l1-2c1-4 2-8 1-12l-4-12-102-178-15-27h-1l-15 27-102 178-4 12a24 24 0 0 0 2 15c4 6 10 12 24 12h190c3 0 18-1 25-12zm-120-178 93 163h-186z"/></g><g transform="matrix(.31945 0 0 .31945 198.01 39.602)"><path d="m74.4 3.7c1 1.1 1.7 2.6 1.6 4.2v81.5c0 1.7-0.6 3.4-1.8 4.7a6.2 6.2 0 0 1-4.6 1.9c-1 0-2-0.2-3-0.5a5 5 0 0 1-2.1-1.4l-52.8-68.7v64.6a5.7 5.7 0 0 1-6 5.9c-1.5 0-3-0.6-4-1.7a5.8 5.8 0 0 1-1.7-4.1v-81.7c0-1.7 0.7-3.4 2-4.6a6 6 0 0 1 4.3-1.8c1 0 2.1 0.2 3 0.6 1 0.3 1.8 1 2.4 1.7l52.5 68.9v-65.3a5.7 5.7 0 0 1 6-5.9c1.5 0 3 0.5 4 1.7zm108.7 0.2c1.1 1.3 1.8 3 1.7 4.6v51.6c0.1 6.6-1.5 13.2-4.7 19-3 5.6-7.5 10.2-13 13.3-5.6 3.3-12 5-18.6 4.9-6.5 0-13-1.6-18.8-4.8a34 34 0 0 1-13-13.2 38.6 38.6 0 0 1-4.7-19.1v-51.6c0-1.7 0.6-3.4 2-4.6a6.7 6.7 0 0 1 9.3 0c1.2 1.2 1.9 2.8 1.9 4.6v51.4c-0.1 4.4 1 8.7 3.2 12.6a23.2 23.2 0 0 0 20.1 11.7 24 24 0 0 0 24.1-24.3v-51.4c0-1.7 0.5-3.4 1.6-4.7 1.2-1.3 2.8-2 4.5-1.9 1.7 0 3.3 0.7 4.4 1.9zm103.9 81.8a6.3 6.3 0 0 1 1.1 3.8c0.1 1.7-0.6 3.4-1.9 4.6a6.4 6.4 0 0 1-4.3 1.7c-2.1 0.1-4.1-1-5.3-2.7l-26-34.8-25.6 34.8a6.2 6.2 0 0 1-5.2 2.8c-1.6 0-3-0.6-4.1-1.7-1-1-1.7-2.4-1.7-3.8 0-1.6 0.6-3.2 1.6-4.4l27.6-37.5-27.7-37.2a6.2 6.2 0 0 1 0.5-8.4 6.3 6.3 0 0 1 7.4-1.2 6 6 0 0 1 2.4 2l25.8 34.6 25.4-34.3c1.1-1.7 3-2.8 5.1-3 1.6 0 3.2 0.8 4.3 2 1.1 1 1.8 2.5 1.9 4 0 1.5-0.5 2.8-1.4 4l-28 37.2 28 37.5zm96.5-82c1.2 1.1 2 2.8 1.9 4.5 0 1.7-0.7 3.3-2 4.4-1.2 1.1-2.9 1.7-4.6 1.6h-23.8v75.4c0 1.7-0.6 3.3-1.9 4.5a7.4 7.4 0 0 1-9.8 0 6.2 6.2 0 0 1-2-4.5v-75.4h-23.8c-1.7 0-3.3-0.5-4.6-1.7a5.9 5.9 0 0 1-1.9-4.5c0-1.6 0.6-3.2 1.8-4.3 1.3-1.2 3-1.8 4.6-1.7h61.4c1.7 0 3.4 0.5 4.7 1.7z"/><path d="m412.7 93.6c-4.3-2.4-8-6-10.5-10.3-0.7-1-1.1-2.3-1.2-3.6 0-1.7 0.7-3.3 2-4.4 1.1-1.2 2.6-1.8 4.2-2 2.4 0.3 4.6 1.5 6 3.4 3 5 8.4 8 14.1 7.8a17.2 17.2 0 0 0 17.7-17.2v-58.8a6 6 0 0 1 2-4.6c1.3-1.2 3-1.9 4.7-1.9 1.8 0 3.5 0.6 4.8 1.8a6.3 6.3 0 0 1 1.8 4.6v59a29 29 0 0 1-15 26 31.2 31.2 0 0 1-30.5 0.2zm92.5 0.4c-5.3-2-10.2-5-14.4-9a7.2 7.2 0 0 1-2.8-5.4 6 6 0 0 1 2-4.4 6 6 0 0 1 4.3-1.9c1.5 0 2.8 0.5 3.9 1.5 3.2 3 7 5.5 11 7.3 4.2 1.6 8.6 2.4 13 2.4 5 0.1 10-1.2 14.3-4 3.5-2 5.8-5.7 5.8-9.8 0.1-4.3-2.1-8.4-5.8-10.8a52 52 0 0 0-17.4-6.3c-9.2-2-16.3-5-21.3-9.3s-7.4-10.3-7.4-17.9c-0.1-5 1.3-9.9 4.2-14 2.9-4 6.8-7.2 11.4-9.2a45.5 45.5 0 0 1 32.4-0.5 33 33 0 0 1 12 7.1c1.8 1.4 3 3.5 3 5.7a6 6 0 0 1-1.8 4.3 5.7 5.7 0 0 1-4.4 2c-1 0-2.2-0.4-3-1.1a32.1 32.1 0 0 0-21.4-8.2c-5-0.3-10 1-14.3 3.6a11.9 11.9 0 0 0 0 19.8 50 50 0 0 0 15.8 6c5.9 1.1 11.6 3 17.1 5.3 4.2 1.9 7.8 4.8 10.3 8.6 2.6 4.4 3.8 9.3 3.6 14.4 0.1 5-1.4 9.8-4.3 13.8-3 4.2-7.2 7.4-12 9.4a49.8 49.8 0 0 1-33.9 0.6z"/></g></g></svg>

```

File: icons/nuxtjs/nuxtjs-plain.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg version="1.1" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path d="m39.267 108.97-0.28372-0.56744c-0.56744-1.1349-0.56744-2.2698-0.28372-3.6883h-30.641l45.395-80.576 19.01 34.33 6.2419-4.5395-19.01-34.33c-0.28372-0.56744-2.5535-3.972-6.2419-3.972-1.7024 0-4.2558 0.56744-6.2419 4.2558l-45.962 81.427c-0.28372 0.85116-2.2698 4.5395-0.56744 7.6605 1.1349 1.7023 2.8372 3.4046 6.8093 3.4046h38.586c-3.9721 0-5.9581-1.7024-6.8093-3.4046zm87.385-7.3767-36.883-66.391c-0.56744-0.56744-2.5535-4.2558-6.2419-4.2558-1.7023 0-4.2558 0.85116-6.2419 4.2558l-4.8232 7.9442v15.321l11.065-19.01 36.6 65.256h-13.903a6.5256 6.5256 0 0 1-0.56743 3.972l-0.28373 0.28373c-1.7023 3.1209-5.9581 3.4046-6.5256 3.4046h21.563c0.85115 0 4.8232-0.28371 6.8093-3.4046 0.85115-1.4186 1.4186-3.9721-0.56744-7.3767zm-20.144 7.3767v-0.28373l0.28374-0.56743c0.2837-1.1349 0.56743-2.2698 0.2837-3.4046l-1.1349-3.4046-28.939-50.502-4.2558-7.6605h-0.28372l-4.2558 7.6605-28.939 50.502-1.1349 3.4046a6.8093 6.8093 0 0 0 0.56744 4.2558c1.1349 1.7023 2.8372 3.4046 6.8093 3.4046h53.907c0.85116 0 5.107-0.28371 7.093-3.4046zm-34.046-50.502 26.386 46.246h-52.772z" stroke-width=".28372"/></svg>
```
Comment (Panquesito7) [inline icons/nuxtjs/nuxtjs-plain.svg:1]:
```suggestion
<svg version="1.1" width="128" height="128" viewBox="0 0 128 128" xmlns="http://www.w3.org/2000/svg"><path d="m39.267 108.97-0.28372-0.56744c-0.56744-1.1349-0.56744-2.2698-0.28372-3.6883h-30.641l45.395-80.576 19.01 34.33 6.2419-4.5395-19.01-34.33c-0.28372-0.56744-2.5535-3.972-6.2419-3.972-1.7024 0-4.2558 0.56744-6.2419 4.2558l-45.962 81.427c-0.28372 0.85116-2.2698 4.5395-0.56744 7.6605 1.1349 1.7023 2.8372 3.4046 6.8093 3.4046h38.586c-3.9721 0-5.9581-1.7024-6.8093-3.4046zm87.385-7.3767-36.883-66.391c-0.56744-0.56744-2.5535-4.2558-6.2419-4.2558-1.7023 0-4.2558 0.85116-6.2419 4.2558l-4.8232 7.9442v15.321l11.065-19.01 36.6 65.256h-13.903a6.5256 6.5256 0 0 1-0.56743 3.972l-0.28373 0.28373c-1.7023 3.1209-5.9581 3.4046-6.5256 3.4046h21.563c0.85115 0 4.8232-0.28371 6.8093-3.4046 0.85115-1.4186 1.4186-3.9721-0.56744-7.3767zm-20.144 7.3767v-0.28373l0.28374-0.56743c0.2837-1.1349 0.56743-2.2698 0.2837-3.4046l-1.1349-3.4046-28.939-50.502-4.2558-7.6605h-0.28372l-4.2558 7.6605-28.939 50.502-1.1349 3.4046a6.8093 6.8093 0 0 0 0.56744 4.2558c1.1349 1.7023 2.8372 3.4046 6.8093 3.4046h53.907c0.85116 0 5.107-0.28371 7.093-3.4046zm-34.046-50.502 26.386 46.246h-52.772z" stroke-width=".28372"/></svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
👍 Good work!

**Comment by TheFoxarmy** [comment]
Hi, I see these changes, and they appear to just make the icons smaller when I preview them. However, going through several other icons, they seem to take up the full size of the page just as mine did before the suggested changes. Why should I use these versions?

**Comment by Panquesito7** [comment]
> they appear to just make the icons smaller when I preview them

Ah, that's because the `width` and the `height` are set to 128. You can remove that, but I recommend doing it that way. 🙂

**Comment by Thomas-Boi** [comment]
Hey @Panquesito7,

We actually don't need the `width` and `height` attributes. It makes styling the SVG harder since it now restricts the SVG forever at 128px if someone uses inline SVG. We want them to scale base on the container that they are in instead. It's fine if the SVG has the `width` and `height` attribute for now. However, the `svg-optimizer` bot will remove it later so it doesn't matter.

I'll update the `CONTRIBUTING.md` so this change is reflected.

File: devicon.json [hunk 5]
```diff
@@ -2840,6 +2840,29 @@
...
+        "tags": [
+            "js",
+            "javascript",
+            "framework",
+            "frontend",
+            "vuejs"
+        ],
+        "color": "#00c48d",
+        "versions": {
+            "SVG": [
```
Comment (Thomas-Boi) [inline devicon.json:2854]:
```suggestion
            "svg": [
```

File: devicon.json [hunk 6]
```diff
@@ -2840,6 +2840,29 @@
...
+                "original",
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        }
```
Comment (Thomas-Boi) [inline devicon.json:2864]:
```suggestion
        },
        "aliases": []
```

**Review by Thomas-Boi — COMMENTED** [review]
Just need to make some update to the `devicon.json`

File: devicon.json [hunk 7]
```diff
@@ -2840,6 +2840,30 @@
...
+    {
+        "name": "nuxtjs",
+        "tags": [
+            "js",
+            "javascript",
+            "framework",
+            "frontend",
+            "vuejs"
+        ],
+        "color": "#00c48d",
```
Comment (Panquesito7) [inline devicon.json:2852]:
```suggestion
```

File: devicon.json [hunk 8]
```diff
@@ -2840,6 +2840,30 @@
...
+                "original-wordmark",
+                "plain",
+                "plain-wordmark"
+            ],
+            "font": [
+                "plain",
+                "plain-wordmark"
+            ]
+        },
+        "aliases": []
```
Comment (Panquesito7) [inline devicon.json:2865]:
```suggestion
        "color": "#00c48d",
        "aliases": []
```

**Review by Panquesito7 — APPROVED** [review]
LGTM. Great work, @TheFoxarmy; thank you! 👍 🎉

**Pull request merged**